app/controllers/home_page_controller.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class HomePageController < ApplicationController
+  def show; end
```

Comment:
**idea**: `show` normally suggests an entity in the database. I have a bias towards `index`. WDYT about this topic?

Reply:
I misunderstood your first comment in regards, and thought I should replace it with show 🤦‍♀️ But, yeah, while I was doing it I thought it was a bit weird to use it this way derp.